api/tests/core/offers/test_api.py

```diff
@@ -2745,7 +2745,7 @@ def test_delete_draft_with_mediation_offer_criterion_activation_code_and_stocks(
 class DeleteStocksTest:
     def test_delete_batch_stocks(self, client):
         stocks = factories.StockFactory.create_batch(3)
-        api.batch_delete_stocks(stocks)
+        api.batch_delete_stocks(stocks, author_id=None, user_connect_as=None)
```

Comment:
Ici et en-dessous, pourquoi passer explicitement `None` lorsque c'est la valeur par défaut ?
```suggestion
        api.batch_delete_stocks(stocks)
```

Reply:
Si je ne le passe pas alors je me prend ce genre d'erreur : 
```
tests/core/offers/test_api.py:2748: [E1120(no-value-for-parameter), DeleteStocksTest.test_delete_batch_stocks] No value for argument 'author_id' in function call
tests/core/offers/test_api.py:2748: [E1120(no-value-for-parameter), DeleteStocksTest.test_delete_batch_stocks] No value for argument 'user_connect_as' in function call
```

erreur que je pourrai résoudre en gardant cela ou en mettant author_id: int | None = None, user_connect_as: bool | None = None
sauf que batch_delete_stocks n'est utilisé que par delete_stocks qui renvoie les arguments à présent et ce ne sera jamais None, j'ai donc trouvé plus sage de les mettre dans les tests plutot que de les mettres en None par default